ui-vue3/src/views/traffic/tagRule/tabs/formView.vue

```diff
@@ -117,7 +117,7 @@
         >
         <a-tag color="#2db7f5">version=v1</a-tag>
       </a-space>
-    </a-card>
+    </a-card >
```

Comment:
Is npm run prettier-format not executed? There are irregularities in some places, such as here

Reply:
get